### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'publicTeams')

### ISSUE THREAD

**Issue by Dschoordsch** [ParabolInc/parabol#9172]
https://parabol.sentry.io/issues/4613481429/?referrer=github_plugin

```
TypeError: Cannot read properties of undefined (reading 'publicTeams')
  at call (./packages/client/components/DashNavList/DashNavList.tsx:77:49)
  at Array.flatMap (<anonymous>)
  at value (./packages/client/components/DashNavList/DashNavList.tsx:1:440)
  at teams (./packages/client/components/DashNavList/DashNavList.tsx:75:96)
  at teams (./packages/client/components/DashNavList/DashNavList.tsx:1:418)
...
(37 additional frame(s) were not displayed)
```

**Comment by Dschoordsch** [comment]
I think this is happening when accepting an invite. We're probably not adding all the required fragments in this case.

**Comment by Dschoordsch** [comment]
I finally found the most common situation where this is triggered:
- the user is either on the organization page (triggers this or #9207) or they're in the activity library which could additionally trigger #9076 
- they then accept an invite in a different tab (or someone accepts their prompt to join org request)
- they're not redirected and we add the new `organization` object from the subscription, which has not all the necessary fields

Solution will be reworking the `AcceptTeamInvitationPayload` subscription a bit to also include the necessary organization fragments. To avoid sending this to all team members, the update should be sent via the notification subscription.

**Comment by Dschoordsch** [comment]
Still happens

**Comment by Dschoordsch** [comment]
I was wrong, it was seen in 7.10, but it's only fixed in 7.11
